### PR TITLE
Server.c: remove dead for() loop used to add test users to the default hub

### PR DESCRIPTION
**Pull request by davidebeatrici**
SoftEther VPN Patch Acceptance Policy:
http://www.softether.org/5-download/src/9.patch

I choose option 1.

### CODE REVIEW

File: src/Cedar/Server.c [hunk 1]
```diff
@@ -2514,6 +2514,8 @@ void SiInitDefaultHubList(SERVER *s)
 	SiSetDefaultLogSetting(&g);
 	SetHubLogSetting(h, &g);
 
+/*
+	// Add test user(s)
```
Comment (moatazelmasry2) [inline src/Cedar/Server.c]: Just remove the whole snippet. It is a dead piece of code anyway.
If we needed this snippet in the future, it is still saved in git history

Comment (davidebeatrici) [reply src/Cedar/Server.c]: Done.

**Comment by moatazelmasry2** [comment]
Merged. Many thanks. I updated the title a bit. I hope this is ok

**Comment by davidebeatrici** [comment]
Yes, I forgot to update it, sorry.

**Pull request merged**